[{"elements": [{"keyword": "Scenario", "name": "A failing test with autoretry", "steps": [{"keyword": "Given", "match": {"arguments": [], "location": "features/steps/test.py:3"}, "name": "we have behave installed", "result": {"duration": 0.00012183189392089844, "status": "passed"}, "line": 16}, {"keyword": "When", "match": {"arguments": [], "location": "features/steps/test.py:7"}, "name": "we implement a test", "result": {"duration": 0.00010704994201660156, "status": "passed"}, "line": 17}, {"keyword": "Then", "match": {"arguments": [], "location": "features/steps/test.py:15"}, "name": "behave will fail it for us...sometimes", "result": {"duration": 0.0002589225769042969, "status": "failed", "error_message": "['Traceback (most recent call last):', '  File /data/home/london/fprincipato/b2c_fix/lib/python2.6/site-packages/behave/model.py, line 1456, in run', '    match.run(runner.context)', '  File /data/home/london/fprincipato/b2c_fix/lib/python2.6/site-packages/behave/model.py, line 1903, in run', '    self.func(context, *args, **kwargs)', '  File features/steps/test.py, line 17, in step_impl', '    assert (1 == 2)', 'AssertionError']"}, "line": 18}], "tags": [{"name": "@autoretry", "line": 14}], "type": "scenario", "line": 15, "uri": "features/test.feature", "description": "", "id": 1}, {"keyword": "Scenario", "name": "A failing test with autoretry", "steps": [{"keyword": "Given", "match": {"arguments": [], "location": "features/steps/test.py:3"}, "name": "we have behave installed", "result": {"duration": 0.00012183189392089844, "status": "passed"}, "line": 16}, {"keyword": "When", "match": {"arguments": [], "location": "features/steps/test.py:7"}, "name": "we implement a test", "result": {"duration": 0.00010704994201660156, "status": "passed"}, "line": 17}, {"keyword": "Then", "match": {"arguments": [], "location": "features/steps/test.py:15"}, "name": "behave will fail it for us...sometimes", "result": {"duration": 0.0002589225769042969, "status": "failed", "error_message": "['Traceback (most recent call last):', '  File /data/home/london/fprincipato/b2c_fix/lib/python2.6/site-packages/behave/model.py, line 1456, in run', '    match.run(runner.context)', '  File /data/home/london/fprincipato/b2c_fix/lib/python2.6/site-packages/behave/model.py, line 1903, in run', '    self.func(context, *args, **kwargs)', '  File features/steps/test.py, line 17, in step_impl', '    assert (1 == 2)', 'AssertionError']"}, "line": 18}], "tags": [{"name": "@autoretry", "line": 14}], "type": "scenario", "line": 15, "uri": "features/test.feature", "description": "", "id": 2}, {"keyword": "Scenario", "name": "A failing test with autoretry", "steps": [{"keyword": "Given", "match": {"arguments": [], "location": "features/steps/test.py:3"}, "name": "we have behave installed", "result": {"duration": 0.00012183189392089844, "status": "passed"}, "line": 16}, {"keyword": "When", "match": {"arguments": [], "location": "features/steps/test.py:7"}, "name": "we implement a test", "result": {"duration": 0.00010704994201660156, "status": "passed"}, "line": 17}, {"keyword": "Then", "match": {"arguments": [], "location": "features/steps/test.py:15"}, "name": "behave will fail it for us...sometimes", "result": {"duration": 0.0002589225769042969, "status": "passed"}, "line": 18}], "tags": [{"name": "@autoretry", "line": 14}], "type": "scenario", "line": 15, "uri": "features/test.feature", "description": "", "id": 3}], "keyword": "Feature", "name": "showing off behave", "tags": [], "line": 1, "uri": "features/test.feature", "description": "", "id": 0}]
